astropy/_erfa/setup_package.py

```diff
@@ -31,9 +32,25 @@ def pre_build_ext_hook(cmd_obj):
 def pre_sdist_hook(cmd_obj):
     preprocess_source()
 
+def is_release():
+    """
+    Determines if we're on a release version of astropy
+    """
```

Comment:
Oh, um, hmm...  Well this is fine for now if it's what it took for this to work.  But it seems to me this kind of information should be more easily made available to `setup_packages` and not require hacks like this (least of all particular to a single module).